<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cebsGparForm</class>
 <widget class="QWidget" name="cebsGparForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GLOBAL PARAMETER</string>
  </property>
  <widget class="QGroupBox" name="groupBox_gpar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>全局参数设置</string>
   </property>
   <widget class="QCheckBox" name="checkBox_gpar_autoIdf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照后自动识别</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gpar_picTti">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gpar_picTti_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>定时拍照间隔</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gpar_picTti_min">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>分钟</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_gpar_autoPic">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>定时自动拍照</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_gpar_picFixPos">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>定点拍照</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_gpar_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>图像识别参数设置</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_gpar_vision_small_low_limit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gpar_picTti_min_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>小尺寸门限</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gpar_vision_small_mid_limit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gpar_picTti_min_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>小-中尺寸门限</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gpar_vision_mid_big_limit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gpar_picTti_min_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>中-大尺寸门限</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gpar_vision_big_upper_limit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gpar_picTti_min_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>大尺寸门限</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_gpar_vision_res_addup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输出图像叠加标定</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_ctrl_compl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>270</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>设定完成</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_eng">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>工程模式命令</string>
   </property>
   <widget class="QTextEdit" name="textEdit_eng_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoAll</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Q01&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eng_send">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eng_clear">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_ctrl_giveup">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>设定放弃</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ctrl_compl</sender>
   <signal>clicked()</signal>
   <receiver>cebsGparForm</receiver>
   <slot>slot_gpar_compl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>875</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ctrl_giveup</sender>
   <signal>clicked()</signal>
   <receiver>cebsGparForm</receiver>
   <slot>slot_gpar_giveup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_calib_close()</slot>
  <slot>slot_calib_move()</slot>
  <slot>slot_calib_left_up()</slot>
  <slot>slot_calib_right_bottom()</slot>
  <slot>slot_calib_pilot()</slot>
  <slot>slot_calib_pilot_stop()</slot>
  <slot>slot_gpar_compl()</slot>
  <slot>slot_gpar_giveup()</slot>
 </slots>
</ui>
